src/components/PopupWithForm.js

```diff
@@ -6,6 +6,13 @@ export default class PopupWithForm extends Popup{
         this._handleFormSubmit = handleFormSubmit;
         this._popupForm = this._popupElement.querySelector('.modal__form');
         this._inputList = this._popupElement.querySelectorAll('.modal__form-input');
+        this._submitBtn = this._popupElement.querySelector('.modal__button');
+        this._submitBtnText = this._submitBtn.textContent;
+        this._profileImage = document.querySelector('.profile__image');
+        this._profileImageButton = document.querySelector('.profile__image-edit');
+        this._modalImage = document.querySelector('#profile');
+        this._modalImageSubmit = this._modalImage.querySelector('#profile-button');
+        this._imageSubmitHandler = this._handleImageSubmit.bind(this);
     }
```

Comment:
Please, delete the hardcode for the avatar from `PopupWithForm`

There should **not** be any code in universal classes that doesn’t suit every popup.